Snowdog_Components/components/Templates/catalog/grid/_grid-variables.scss

```diff
@@ -0,0 +1,8 @@
+// Catalog Grid
+$catalog-grid__gap                      : $spacer--medium !default;
+$catalog-grid__margin                   : 0 !default;
+$catalog-grid__padding                  : $spacer--medium 0 $spacer--large 0 !default;
+$catalog-grid__column-base-fraction     : 1fr !default;
```

Comment:
I can't think of a case where we would need to change this value 🤔 I think it's overkill and we can use `1fr` in column-related variables. or maybe you have an idea of when it would be useful?

Reply:
Only one idea came to my mind, when we'd like to set a fixed value for column. Hovewer, using variables in this case might be an overkill. Removing that.